test/System.Web.Http.Integration.Test/Tracing/TracingTest.cs

```diff
@@ -397,8 +409,8 @@ private static IList<string> UnexpectedTraces(IList<ExpectedTraceRecord> expecte
 
                 if (expectedTrace == null)
                 {
-                    unexpected.Add(string.Format("kind={0} category={1}, operator={2}, operation={3}",
-                                    actualRecord.Kind, actualRecord.Category, actualRecord.Operator, actualRecord.Operation));
+                    unexpected.Add(string.Format("kind={0} category={1}, operator={2}, operation={3}, message={4}",
+                        actualRecord.Kind, actualRecord.Category, actualRecord.Operator, actualRecord.Operation, actualRecord.Message));
```

Comment:
Nit;

```suggestion
                    unexpected.Add($"kind={actualRecord.Kind} category={actualRecord.Category}, operator={actualRecord.Operator}, operation={actualRecord.Operation}, message={actualRecord.Message}");
```

Reply:
We're using an old version of C# in this repo, `<LangVersion>5</LangVersion>` in particular. `string` interpolation is not supported.